--- a/usr/local/etc/inc/system.inc	2024-11-28 23:35:34.035057000 +0300
+++ b/usr/local/etc/inc/system.inc	2024-11-28 19:36:23.410236000 +0300
@@ -1080,7 +1080,7 @@
     }
 
     foreach ($mods as $mod) {
-        mwexecf('/sbin/kldload %s', $mod, true);
+        mwexecf('if [ "`kenv %s_load`" != NO ]; then /sbin/kldload %s; fi', array($mod, $mod), true);
     }
 
     /* we now have /dev/pf, time to fix permissions for proxies */
